README.md

```diff
@@ -1,23 +1,23 @@
 
 # Group_E_Snakes_and_Ladders
 
-We will be simulating game of snakes and ladders the using 
+We will be simulating game of snakes and ladders using the
 C Programming language. 
 
 # Organisation
 Carleton University
 
 # Authors
-- Vidushi Gupta Vidushigupta@cmail.carleton.ca
 - Diksha Saini Dikshasaini@cmail.carleton.ca
 - Sayali Dhokar Sayalidhokar@cmail.carleton.ca
 - Rupneet Kaur Gill Rupneetkaurgill@cmail.carleton.ca
+- Vidushi Gupta Vidushigupta@cmail.carleton.ca
 
 # Brief Description
 
 Below are the rules of the game under which we will design an application.
 1. Snakes and ladders game is a grid of 10 x10 squares. The entire grid is numbered from 1 to 100.
-2. *It is a multi-player game. 2 or more players are required.*
+2. *It is a multi-player game. 2 players are required.*
 3. Each player takes turns to roll a six-sided dice. The player moves as many squares forward as     number on the dice. A player gets a consecutive turn if player gets a six.
 4. Each player starts at 1. Each player needs to roll 1 or a 6 to start playing the game.
 5. If a player’s counter lands at bottom of ladder, the player moves to top of ladder.
```

Comment:
Similarly, if players counter lands on the top of the snake, the player moves to the bottom of the snake.

Reply:
yes